### DISCUSSION TITLE
Can Ionide run inside a dev container?

### DISCUSSION THREAD

**Discussion by dlidstrom** [ionide/ionide-vscode-fsharp#1818] [Q&A] [answered]
I'm working on macos where Ionide/f# language server doesn't function as of currently (I see similar errors as in #1793).
Is it possible to run Ionide in a dev container? If so, does anyone have a sample `devcontainer.json` file to share?

**Comment by TheAngryByrd** [comment, marked as answer]
Yeah, I have it running in [FsToolkit.ErorrHandling](https://github.com/demystifyfp/FsToolkit.ErrorHandling/tree/master/.devcontainer).

**Reply by baronfel** [reply]
Also, they have it configured for the F# compiler: [dotnet/fsharp](https://github.com/dotnet/fsharp/tree/main/.devcontainer).

**Reply by dlidstrom** [reply]
Cool! Thanks for sharing.
